src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -157,45 +157,23 @@
+                        " - If running in any other running mode: when a group which is being crosschecked does not have any observations at fingerprinting sites, a warning will be logged. \n" +
                         "\n" +
-                        "There are cases where one would like to identify a few groups out of a collection of many possible groups (say " +
-                        "to link a SAM to its correct sample in a multi-sample VCF. In this case one would not case for the cross-checking " +
-                        "of the various samples in the VCF against each other, but only in checking the identity of the SAM against the various " +
-                        "samples in the VCF. The SECOND_INPUT is provided for this use-case. With SECOND_INPUT provided, CrosscheckFingerprints " +
-                        "does the following:\n" +
-                        " - aggregation of data happens independently for the input files in INPUT and SECOND_INPUT. \n" +
-                        " - aggregation of data happens at the SAMPLE level \n" +
-                        " - each samples from INPUT will only be compared to that same sample in SECOND_INPUT. \n" +
-                        " - MATRIX_OUTPUT is disabled. " +
-                        "\n" +
-                        "In some cases, the groups collected may not have any observations (calls for a VCF, reads for a SAM) at fingerprinting sites, or " +
-                        "a sample in INPUT may be missing from the SECOND_INPUT. These cases are handled as follows:  If running in CHECK_SAME_SAMPLES mode " +
-                        "with INPUT and SECOND_INPUT, and either INPUT or SECOND_INPUT includes a sample not found in the other, or contains a sample with " +
-                        "no observations at any fingerprinting sites, an error will be logged and the tool will return EXIT_CODE_WHEN_MISMATCH. In all other " +
-                        "running modes, when any group which is being crosschecked does not have any observations at fingerprinting sites, a warning is " +
-                        "logged.  As long as there is at least one comparison where both sides have observations at fingerprinting sites, the tool will " +
-                        "return zero.  However, if all comparisons have at least one side with no observations at fingerprinting sites, an error will be " +
-                        "logged and the tool will return EXIT_CODE_WHEN_NO_VALID_CHECKS." +
+                        "Note that, as long as there is at least one comparison in which both files have observations at fingerprinting sites, the tool will return a ‘zero’. However, an error will be logged and the tool will return EXIT_CODE_WHEN_NO_VALID_CHECKS if all comparisons have at least one side without observations at a fingerprinting site (ie. all LOD scores are zero). \n" +
```

Comment:
here too, as in other places, I think that a long line of text is harder to read in code...please revert this back to multiple lines....